confignode/src/main/java/org/apache/iotdb/confignode/conf/ConfigNodeStartupCheck.java

```diff
@@ -405,6 +427,119 @@ private void createDirIfEmpty(File dir) throws IOException {
     }
   }
 
+  /** Check configuration */
+  public boolean checkConfigurations(TConsensusGroupId groupId) {
```

Comment:
Merge with checkSystemProperties() in this class

Reply:
> Merge with checkSystemProperties() in this class

checkSystemProperties() check configurations of itself, but checkConfigurations() check each other between other seed-node.